modules/bundles/get_started.ipynb

```diff
@@ -164,8 +164,6 @@
     "        \"$import os\",\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">A MONAI bundle supports flexible customized logic, there are several ways to achieve this:

If defining own components like transform, loss, trainer, etc. in a python file, just use its module path in _target_ within the config file.
Parse the config in your own python program and do lazy instantiation with customized logic.

Here we show an example to parse the config in python code and execute the training.
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Project-MONAI/tutorials/pull/673/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='37'/>

Reply:
Sure, updated in the latest commit.

Thanks.